ui/tests/integration/components/selectable-card-test.js

```diff
@@ -0,0 +1,30 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+const TOTAL = 15;
+const CARD_TITLE = 'Tokens';
+
+module('Integration | Component meep', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.set('total', TOTAL);
+    this.set('cardTitle', CARD_TITLE);
+  });
+
+  test('it shows the card title', async function(assert) {
```

Comment:
nit: if this is asserting that the proper `total` is shown (i.e. 15) and not `cardTitle` ("Tokens"), the test message here should be `'it shows the card total'`.

Reply:
Got it.  Because the "total" acts like a card title (it has the largest font size on the component) in my mind it was the title, but I think you're right, total is better.